src/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  outline: 0 !important;
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+  border-radius: 4px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form:focus {
+  background: linear-gradient(180deg, #fff, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input {
```

Comment:
input should have 100% width
![image](https://user-images.githubusercontent.com/37161315/154444568-023eb81a-13ac-484f-bb43-f2c61135b770.png)